config/routes.rb

```diff
@@ -1,12 +1,16 @@
 Rails.application.routes.draw do
-  devise_for :users
-  root "users#index"
+  devise_for :users,
+    controllers: {
+      sessions: 'users/sessions',
+      registrations: 'users/registrations'
+    }
```

Comment:
- [ ] I am not able to register a new user. This is because you are using GET request instead of POST when registering a new user. Screenshot from routes:
![Screenshot from 2022-02-05 17-21-14](https://user-images.githubusercontent.com/20245164/152645773-631a6aed-38b7-4db9-9e63-ef3a2ccfd29f.png)

- [ ] Could you please include the API endpoints in the readme so as to make it easier for the users to test them.

Reply:
done